Unit test test_resend_stored_flows  failing

```
============================= test session starts ==============================
platform linux -- Python 3.9.13, pytest-7.0.0, pluggy-1.0.0
cachedir: .tox/py39/.pytest_cache
rootdir: /builds/kytos/flow_manager/flow_manager
plugins: cov-3.0.0
collected 87 items

tests/integration/test_main.py ..                                        [  2%]
tests/unit/test_barrier_request.py .                                     [  3%]
tests/unit/test_flow_controller.py ..........                            [ 14%]
tests/unit/test_flow_serializer_v0x01.py ....                            [ 19%]
tests/unit/test_flow_serializer_v0x04.py ..........                      [ 31%]
tests/unit/test_main.py ........................F................        [ 78%]
tests/unit/test_match_13.py ...........                                  [ 90%]
tests/unit/test_utils.py ........                                        [100%]

=================================== FAILURES ===================================
______________________ TestMain.test_resend_stored_flows _______________________

self = <flow_manager.tests.unit.test_main.TestMain testMethod=test_resend_stored_flows>
mock_install_flows = <MagicMock name='_install_flows' id='140042366073680'>

    @patch("napps.kytos.flow_manager.main.ENABLE_CONSISTENCY_CHECK", False)
    @patch("napps.kytos.flow_manager.main.Main._install_flows")
    def test_resend_stored_flows(self, mock_install_flows):
        """Test resend stored flows."""
        dpid = "00:00:00:00:00:00:00:01"
        switch = get_switch_mock(dpid, 0x04)
        mock_event = MagicMock()
        self.napp.flow_controller.get_flows.return_value = [MagicMock()]
    
        mock_event.content = {"switch": switch}
        self.napp.controller.switches = {dpid: switch}
        self.napp.resend_stored_flows(mock_event)
>       mock_install_flows.assert_called()

tests/unit/test_main.py:484: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <MagicMock name='_install_flows' id='140042366073680'>

    def assert_called(self):
        """assert that the mock was called at least once
        """
        if self.call_count == 0:
            msg = ("Expected '%s' to have been called." %
                   (self._mock_name or 'mock'))
>           raise AssertionError(msg)
E           AssertionError: Expected '_install_flows' to have been called.

/usr/local/lib/python3.9/unittest/mock.py:876: AssertionError
------------------------------ Captured log call -------------------------------

```

--- a/main.py
+++ b/main.py
@@ -104,7 +104,11 @@ class Main(KytosNApp):
         log.debug("flow-manager stopping")
 
     @listen_to("kytos/of_core.handshake.completed")
-    def resend_stored_flows(self, event):
+    def on_resend_stored_flows(self, event):
+        """Resend stored Flows."""
+        self.resend_stored_flows(event)
+
+    def resend_stored_flows(self, event) -> None:
         """Resend stored Flows."""
         # if consistency check is enabled, it should take care of this
         if ENABLE_CONSISTENCY_CHECK:

